lib/archive_importer/own_entity_importer.rb

```diff
@@ -25,7 +25,7 @@ def persisted_object
     end
 
     def real_author
-      instance.author.diaspora_handle
+      instance.try(:author).try(:diaspora_handle)
```

Comment:
By the way: Since we support only ruby 2.3 and newer, you can use the new [Safe Navigation Operator](http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/) `&.` here.